tests/backend_test.py

```diff
@@ -1101,7 +1104,7 @@ def test_postprocess_emu(manila_emulator_backend: IBMQEmulatorBackend) -> None:
     assert sum(counts.values()) == 10
 
 
-@pytest.mark.timeout(None)
+@pytest.mark.flaky(reruns=3, reruns_delay=2)
```

Comment:
Do you think 2 is a good delay? I thought it should be longer, but I am not sure. Have you tried if this helps?

Reply:
The tests seem to pass with a delay of 2 seconds so far. If we still get failures we can up this to 5 or 10

Reply:
Got a failure
https://github.com/CQCL/pytket-qiskit/actions/runs/5321790504/jobs/9637347806
I'll try 10 seconds instead of 2